aroma/utils.py

```diff
@@ -40,135 +39,7 @@ def cross_correlation(a, b):
-                ),
-            },
-        }
-
-    # Classify the ICs as motion (rejected) or non-motion (accepted)
-    features_df["classification"] = "accepted"
-    features_df["rationale"] = ""
-
-    # CSF
-    rej_csf = features_df["csf_fract"] > THR_CSF
-    features_df.loc[rej_csf, "classification"] = "rejected"
-    features_df.loc[rej_csf, "rationale"] += "CSF;"
-
-    # HFC
-    rej_hfc = features_df["HFC"] > THR_HFC
-    features_df.loc[rej_hfc, "classification"] = "rejected"
-    features_df.loc[rej_hfc, "rationale"] += "HFC;"
-
-    # Hyperplane
-    # Project edge & max_RP_corr feature scores to new 1D space
-    x = features_df[["max_RP_corr", "edge_fract"]].values
-    proj = HYPERPLANE[0] + np.dot(x, HYPERPLANE[1:])
-    rej_hyperplane = proj > 0
-    features_df.loc[rej_hyperplane, "classification"] = "rejected"
-    features_df.loc[rej_hyperplane, "rationale"] += "hyperplane;"
-
-    # Classify the ICs
-    is_motion = (features_df["csf_fract"] > THR_CSF) | (features_df["HFC"] > THR_HFC) | (proj > 0)
-    assert np.array_equal(is_motion, (features_df["classification"] == "rejected").values)
-
-    # Reorder columns and remove trailing semicolons
-    features_df = clean_dataframe(features_df)
-
-    return features_df, metric_metadata
-
-
-def write_metrics(features_df, out_dir, metric_metadata=None):
-    """Write out feature/classification information and metadata.
-
-    Parameters
-    ----------
-    features_df : (C x 5) :obj:`pandas.DataFrame`
-        DataFrame with metric values and classifications.
-        Must have the following columns: "edge_fract", "csf_fract", "max_RP_corr", "HFC", and
-        "classification".
-    out_dir : :obj:`str`
-        Output directory.
-    metric_metadata : :obj:`dict` or None, optional
-        Metric metadata in a dictionary.
-
-    Returns
-    -------
-    motion_ICs : array_like
-        Array containing the indices of the components identified as motion components.
-
-    Output
-    ------
-    AROMAnoiseICs.csv : A text file containing the indices of the
-                        components identified as motion components
-    desc-AROMA_metrics.tsv
-    desc-AROMA_metrics.json
-    """
-    # Put the indices of motion-classified ICs in a text file (starting with 1)
-    motion_ICs = features_df["classification"][features_df["classification"] == "rejected"].index
-    motion_ICs = motion_ICs.values
-
-    with open(op.join(out_dir, "AROMAnoiseICs.csv"), "w") as fo:
-        out_str = ",".join(motion_ICs.astype(str))
-        fo.write(out_str)
-
-    # Create a summary overview of the classification
-    out_file = op.join(out_dir, "desc-AROMA_metrics.tsv")
-    features_df.to_csv(out_file, sep="\t", index_label="IC")
-
-    if isinstance(metric_metadata, dict):
-        with open(op.join(out_dir, "desc-AROMA_metrics.json"), "w") as fo:
-            json.dump(metric_metadata, fo, sort_keys=True, indent=4)
-
-    return motion_ICs
```

Comment:
Why do we want to remove this?

Reply:
This was very long ago, but it seems to me that the plan would be to write the outputs somewhere else, when we have more clarity on what we want to exactly write out.

Reply:
Ok, I will create `io.py` and put it in there.